src/nvim/popupmnu.c

```diff
@@ -136,7 +136,6 @@ void pum_display(pumitem_T *array, int size, int selected, bool array_changed,
       } else {
         ui_call_popupmenu_select(selected);
       }
-      return;
```

Comment:
why remove this `return` ?

Reply:
I removed this `return;` so that `pum_height` does not become 0 when UI does not call `nvim_pum_set_height()`. For now, I moved `return;` right after `ui_call_popupmenu_select()`.

If the UI does not call `nvim_pum_set_height()`, the value calculated from the line of the cursor and the number of lines on the screen is used as `pum_height`. I think this is better than 0 or a fixed value of 10.

Reply:
Sure, makes sense.